conda/environments/triton_test_no_client.yml

```diff
@@ -0,0 +1,19 @@
+name: triton_test
+channels:
+  - conda-forge
+  - nvidia
+  - rapidsai
+dependencies:
+  - aws-sdk-cpp<1.9
+  - clang-tools=11.1.0
+  - cudatoolkit=11.4
+  - cuml=21.12
+  - flake8
+  - hypothesis
+  - lightgbm
+  - pip
+  - pytest
+  - python=3.8
+  - rapidsai::py-xgboost
```

Comment:
Any particular reason to prefer py-XGBoost over the regular XGBoost package?

Reply:
Nope. Come to think of it, I'm a little fuzzy on the difference between those packages... Time to check the docs.